resources/content/schema/user/user-auth.json

```diff
@@ -117,6 +117,14 @@
         "port.publicIpAddressId" : "r",
         "port.publicPort" : "ru",
 
+        "project" : "crud",
```

Comment:
I believe delete does not work, don't allow it.  Please add test for auth in test_authorization.py